src/intl/en/learn-quizzes.json

```
@@ -1,14 +1,33 @@
 {
+  "add-quiz": "Add a question/quiz",
+  "average-score": "Average score:",
+  "basics": "Ethereum Basics",
+  "basics-description": "This section covers the fundamental concepts of Ethereum, ensuring you have a strong foundation.",
+  "completed": "Completed:",
+  "community-stats": "Community stats",
+  "contribute": "Contribute to our library.",
   "correct": "Correct",
   "explanation": "Explanation",
   "next-question": "Next question",
+  "next-quiz": "Next quiz",
+  "passed": "You passed the quiz!",
+  "questions": "Questions",
+  "questions-answered": "Questions answered:",
+  "quizzes-subtitle": "Find out how well you understand Ethereum and cryptocurrencies. Are you ready to become an expert?",
+  "retry": "Retry rate:",
   "score": "Score",
   "see-results": "See results",
   "share-results": "Share results",
+  "start": "Start",
   "submit-answer": "Submit answer",
-  "test-your-knowledge": "Test your knowledge",
+  "test-your-knowledge": "Test Your Ethereum Knowledge",
```

We use sentence case as a convention (cc: @konopkja )

```suggestion
  "test-your-knowledge": "Test your Ethereum knowledge",
```